lesson01.calculator/index.html

```diff
@@ -6,17 +6,146 @@
+        <button class="calc_button" id="" onclick="actionclicked('^');" style="grid-area: pow;">^</button>
+        <button class="calc_button" id="" onclick="numPadPi();" style="grid-area: pi;">&#x1D745;</button>
+        <button class="calc_button" id="" onclick="numPadEnter('1');" style="grid-area: one;">1</button>
+        <button class="calc_button" id="" onclick="numPadEnter('2');" style="grid-area: two;">2</button>
+        <button class="calc_button" id="" onclick="numPadEnter('3');" style="grid-area: thr;">3</button>
+        <button class="calc_button" id="" onclick="numPadEnter('4');" style="grid-area: fou;">4</button>
+        <button class="calc_button" id="" onclick="numPadEnter('5');" style="grid-area: fiv;">5</button>
+        <button class="calc_button" id="" onclick="numPadEnter('6');" style="grid-area: six;">6</button>
+        <button class="calc_button" id="" onclick="numPadEnter('7');" style="grid-area: sev;">7</button>
+        <button class="calc_button" id="" onclick="numPadEnter('8');" style="grid-area: eig;">8</button>
+        <button class="calc_button" id="" onclick="numPadEnter('9');" style="grid-area: nin;">9</button>
+        <button class="calc_button" id="" onclick="numPadEnter('0');" style="grid-area: zer;">0</button>
+        <button class="calc_button" id="" onclick="numPadDot();" style="grid-area: p;">.</button>
+        <button class="calc_button" id="" onclick="actionclicked('=');" style="grid-area: e;">=</button>
+        <button class="calc_button" id="" onclick="actionclicked('+');" style="grid-area: a;">+</button>
+        <button class="calc_button" id="" onclick="actionclicked('-');" style="grid-area: r;">-</button>
+        <button class="calc_button" id="" onclick="actionclicked('/');" style="grid-area: d;">&#xF7;</button>
+        <button class="calc_button" id="" onclick="actionclicked('*');" style="grid-area: m;">&#xd7;</button>
     </main>
+
+    <script>
+            var totalNum, currentNum, calcAction,ans;
+            var calcScreen = document.getElementById("calc_screen");
+            clearCalc();
+    
+            function printValue(val){
+                calcScreen.innerHTML=val;
+                return;
+            }
+            function numPadEnter(digit){
+                currentNum+=digit;
+                printValue(currentNum);
+                return;
+            }
+            function numPadDot(){
+                if(!currentNum.includes(".")){
+                    currentNum+=(currentNum==""?"0":"");
+                    numPadEnter(".");
+                    return;
+                }
+                return;
+            }
+            function clearCalc(){
+                totalNum=0;
+                ans=0;
+                currentNum="";
+                calcAction="+";
+                printValue("0");
+                return;
+            }
+            function numPadPi(){
+                currentNum="3.14159265358979323";
```

Comment:
Nice that you added pai! :+1: 
Now, check [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI) out